src/bootstrap/src/utils/change_tracker.rs

```diff
@@ -390,4 +390,9 @@ pub const CONFIG_CHANGE_HISTORY: &[ChangeInfo] = &[
         severity: ChangeSeverity::Warning,
         summary: "The default configuration filename has changed from `config.toml` to `bootstrap.toml`. `config.toml` is deprecated but remains supported for backward compatibility.",
     },
+    ChangeInfo {
```

Comment:
This is a huge change, which IMO requires a much more thorough explanation to rustc contributors. We should provide a link to some resource that will clearly describe what has changed and what implications it has. The current description of this PR contains almost no information, and links to an old MCP that describes changes that were already made, but also some things that weren't made yet, and mentions things like `--target-sysroot` that don't exist in bootstrap currently.

I think that expanding the description of this PR to include a more comprehensive explanation of what changes would be enough.

Reply:
FWIW, I think we should have a dedicated Inside Rust blog post to announce this (I'm happy to help write one, just haven't quite caught up to the changes here yet).

Reply:
Yeah, or that :)